### PR TITLE
feat(blob): add blob download url

### PR DESCRIPTION
**Pull request by luismeyer**
needs https://github.com/vercel/infra/pull/6667 and https://github.com/vercel/edge-functions/pull/1249

### CODE REVIEW

**Review by vvo — COMMENTED** [review]
Does this PR already includes changes from our API where downloadUrl is sent as the blob response? I believe we need to upgrade the API_VERSION too?

**Comment by luismeyer** [comment]
> Does this PR already includes changes from our API where downloadUrl is sent as the blob response? I believe we need to upgrade the API_VERSION too?

why do we need a new API_VERSION. Technically it works for all versions. For older blobs the url and downloadUrl will behave the same but that's ok isn't it?

File: packages/blob/src/index.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import type { PutCommandOptions } from './put';
 import { createPutMethod } from './put';
 
-// expose generic BlobError
-export { BlobError } from './helpers';
+// expose generic BlobError and download url util
+export { BlobError, getDownloadURL } from './helpers';
```
Comment (vvo) [inline packages/blob/src/index.ts]: Let's never uppercase fully URL, let's only use camelCaseUrl everywhere (public and private APIs)

Comment (luismeyer) [reply packages/blob/src/index.ts]: done

**Comment by vvo** [comment]
@luismeyer so we do not forget:
- [x] document getDownloadUrl in CHANGELOG
- [ ] document getDownloadUrl and downloadUrl on website

**Review by vvo — APPROVED** [review]

**Pull request merged**